phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/application/skin/ApplicationInformationPanelSkin.java

```diff
@@ -198,7 +202,29 @@ private void updateScripts(final GridPane scriptGrid) {
                 }
             });
 
-            scriptGrid.addRow(i, scriptName, installButton);
+            OperatingSystem curOs = new OperatingSystemFetcher().fetchCurrentOperationSystem();
+            Label lTesting = new Label();
+            if (script.getTestingOperatingSystems().contains(curOs)) {
+                assignIcon(lTesting, "Testing", "testingIcon", 30);
```

Comment:
Now that we are using CSS to set the images I'm not sure whether this method is still necessary, because it contains only 3 lines of code.

I think it is a better idea to replace it with the code inside the method:

```suggestion
                lTesting.getStyleClass().add("testingIcon");
                lTesting.setTooltip(new Tooltip(tr(Testing")));
                lTesting.setMinSize(30, 30);
```